cucumber-rails.gemspec

```
@@ -9,10 +9,19 @@ Gem::Specification.new do |s|
   s.description = 'Cucumber Generator and Runtime for Rails'
   s.summary     = "#{s.name}-#{s.version}"
   s.email       = 'cukes@googlegroups.com'
-  s.homepage    = 'http://cukes.info'
+  s.homepage    = 'https://cucumber.io'
 
   s.license = 'MIT'
 
+  s.metadata = {
+    'bug_tracker_uri' => 'https://github.com/cucumber/cucumber-rails/issues',
+    'changelog_uri' => "https://github.com/cucumber/cucumber-rails/blob/v#{s.version}/CHANGELOG.md",
+    'documentation_uri' => 'https://cucumber.io/docs',
+    'mailing_list_uri' => 'https://groups.google.com/forum/#!forum/cukes',
+    'source_code_uri' => "https://github.com/cucumber/cucumber-rails/tree/v#{s.version}",
+    'wiki_uri' => 'https://github.com/cucumber/cucumber-rails/wiki'
```

This one should perhaps be omitted - we are in the process of moving hard-to-keep-current wiki content to the docs site.

---

I notice there's an (old and non-working) value recorded on [Rubygems](https://rubygems.org/gems/cucumber-rails):

http://wiki.github.com/cucumber/cucumber-rails

I wonder if setting this to `nil` would remove it.

```suggestion
    'wiki_uri' => nil
```

---

I notice there's an (old and non-working) value recorded on [Rubygems](https://rubygems.org/gems/cucumber-rails):

http://wiki.github.com/cucumber/cucumber-rails

I wonder if setting this to `nil` would remove it.

```suggestion
    'wiki_uri' => nil
```

---

The absence of the `'wiki_uri` key would be best.

---

Removed in 0b036ea